entity-framework/core/providers/provider-log.md

```diff
@@ -18,85 +18,86 @@ We are starting this log with changes from 2.1 to 2.2. Prior to 2.1 we used the
 
 Note that many of the [application-level breaking changes](../what-is-new/ef-core-3.0/breaking-changes.md) will also impact providers.
 
-* https://github.com/aspnet/EntityFrameworkCore/pull/14022
+* <https://github.com/aspnet/EntityFrameworkCore/pull/14022>
```

Comment:
Not a fan

Reply:
Doesn't bother me either way, I just like to follow orders...

Reply:
The rule here has a good explanation of why we do this:
https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md

This is to be clear about the intent.